brainstormforce/ultimate-addons-for-gutenberg
# Fix: Blocks throwing an error when deactivate all the UAG blocks except the `HowTo Schema` and `Multi Buttons`.

### Description

Trello - https://trello.com/c/npBgSX79/288-fix-multi-button-throwing-an-error-when-deactivate-all-the-uag-blocks-except-the-multi-buttons

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Deactivate all blocks except multi buttons.
- Create a new page an drag and drop multi-button on page. It throws an error.
- Switch to `multi-button-is-child-issue` and drag and drop block again and check.

### Checklist:
